src/Api/Dfe.Complete.Api/Controllers/ProjectsController.cs

```diff
@@ -347,6 +350,97 @@ public async Task<IActionResult> ListAllProjectsByTrustRefAsync([FromQuery] List
+        {
+            var result = await sender.Send(request, cancellationToken);
+
+            if (!result.IsSuccess || result.Value is null)
+                return NotFound();
+
+            return Ok(result.Value);
+        }
+
+        /// <summary>
+        /// Create a new Note for a Project
+        /// </summary>
+        /// <param name="request">The request.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        [Authorize(Policy = "CanReadWrite")]
+        [HttpPost]
+        [Route("Notes")]
+        [SwaggerResponse(201, "Note ID", typeof(NoteId))]
+        [SwaggerResponse(404, "Note not found")]
+        public async Task<IActionResult> CreateProjectNoteAsync([FromBody] CreateNoteCommand request, CancellationToken cancellationToken)
+        {
+            var result = await sender.Send(request, cancellationToken);
+
+            if (!result.IsSuccess || result.Value is null)
+                return StatusCode(500, result.Error);
+
+            return Created("", new NoteId(result.Value.Value));
+        }
+
+        /// <summary>
+        /// Update a Note for a Project
+        /// </summary>
+        /// <param name="request">The request.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        [Authorize(Policy = "CanReadWriteUpdate")]
+        [HttpPut]
+        [Route("Notes")]
+        [SwaggerResponse(200, "Note ID", typeof(NoteId))]
+        [SwaggerResponse(401, "Unauthorized access")]
+        [SwaggerResponse(404, "Note not found")]
+        public async Task<IActionResult> UpdateProjectNoteAsync([FromBody] UpdateNoteCommand request, CancellationToken cancellationToken)
+        {
+            var result = await sender.Send(request, cancellationToken);
+
+            if (!result.IsSuccess || result.Value is null)
+            {
+                if (result.ErrorType == ErrorType.Unauthorized)
+                    return Unauthorized(result.Error);
+                return StatusCode(500, result.Error);
+            }
+
+            return Ok(new NoteId(result.Value.Value));
+        }
+
+        /// <summary>
+        /// Delete a Note for a Project
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        [Authorize(Policy = "CanReadWriteUpdateDelete")]
+        [HttpDelete]
+        [Route("Notes")]
+        [SwaggerResponse(200, "Success", typeof(bool))]
+        [SwaggerResponse(404, "Project not found")]
+        [SwaggerResponse(404, "Note not found")]
+        public async Task<IActionResult> DeleteProjectNoteAsync([FromBody] RemoveNoteCommand request, CancellationToken cancellationToken)
+        {
+            var result = await sender.Send(request, cancellationToken);
+
+            if (!result.IsSuccess)
```

Comment:
Same - (ignore it if want to keep multiple if conditions)
```suggestion
           if (!result.IsSuccess)
    return result.ErrorType == ErrorType.Unauthorized
        ? Unauthorized(result.Error)
        : StatusCode(500, result.Error);

```